app/router.scrollBehavior.js

```diff
@@ -1,5 +1,19 @@
+const findEl = async (hash, x = 0) => {
+  return (
+    document.querySelector(hash) ||
+    new Promise((resolve) => {
+      if (x > 50) {
+        return resolve(document.querySelector('#app'))
+      }
+      setTimeout(() => {
+        resolve(findEl(hash, ++x || 1))
+      }, 100)
```

Comment:
is it necessary for this `setTimeout`?

Reply:
@floyd-li Yes it seems like it's used for when you load the page with an anchor tag in the url. I got this fix from the nuxt github repo so i'll trust them this time 🤣

Reply:
Also the links on your policy-page don't scroll so you may require the fix as well.

Reply:
> Also the links on your privacy-policy page don't scroll so you may require the fix as well.

yes for sure. just add your code and test it locally it works well 👍